src/Chapter1.hs

```diff
@@ -489,8 +487,8 @@ Implement a function that returns the last digit of a given number.
   results. Or you can try to guess the function name, search for it and check
   whether it works for you!
 -}
--- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit n = n `mod` 10
```

Comment:
Nice work on using infix form of calling functions! We don't explain this syntax in our course to make it short. But well done :+1: 

Unfortunately, this implementation returns negative numbers on negative inputs because of how `mod` works. Sometimes corner cases can be tricky to spot and fix...